<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegistrationWindow</class>
 <widget class="QWidget" name="RegistrationWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>640</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>440</width>
    <height>580</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Окно Регистрации</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>12</number>
   </property>
   <property name="leftMargin">
    <number>28</number>
   </property>
   <property name="topMargin">
    <number>28</number>
   </property>
   <property name="rightMargin">
    <number>28</number>
   </property>
   <property name="bottomMargin">
    <number>28</number>
   </property>
   <item>
    <widget class="QLabel" name="registerLabel">
     <property name="font">
      <font>
       <pointsize>26</pointsize>
       <weight>75</weight>
      </font>
     </property>
     <property name="text">
      <string>Регистрация</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_1">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>

   <!-- Поля ввода -->
   <item>
    <widget class="QLabel" name="fullNameLabel">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Полное имя</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="fullNameLine">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
    </widget>
   </item>

   <item>
    <widget class="QLabel" name="emailLabel">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>email</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="emailLine">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
    </widget>
   </item>

   <item>
    <widget class="QLabel" name="loginLabel">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Логин</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="loginLine">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
    </widget>
   </item>

   <item>
    <widget class="QLabel" name="passwordLabel">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Пароль</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="passwordLine">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </item>

   <item>
    <widget class="QLabel" name="confirmPasswordLabel">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Подтверждение пароля</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="confirmPasswordLine">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </item>

   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>

   <item>
    <widget class="QPushButton" name="registerButton">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>44</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Зарегистрироваться</string>
     </property>
    </widget>
   </item>

   <item>
    <widget class="QPushButton" name="authorizationLink">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Уже есть аккаунт? Войти здесь.</string>
     </property>
    </widget>
   </item>

  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>